docs/examples/aadpodidentity/aadpodidbinding.yaml

```diff
@@ -1,8 +0,0 @@
-# Please see https://github.com/Azure/aad-pod-identity for more inromation
```

Comment:
Did you deliberately mean to delete these?

Reply:
yes!